### DISCUSSION TITLE
Lets put ufw instructions in the wiki

### DISCUSSION THREAD

**Discussion by JavaMiata** [dtcooper/raspotify#613] [General]
I just set up raspotify and found that I needed to poke a few holes in my firewall, and ended up finding this issue: https://github.com/dtcooper/raspotify/issues/139
I think we ought to put these instructions in the basic setup guide
set the zeroconf port: 
LIBRESPOT_ZEROCONF_PORT="65444"
add the ports:
sudo ufw allow from 192.168.0.0/16 to any port 5353 proto udp
sudo ufw allow from 192.168.0.0/16 to any port 65444 proto tcp

**Comment by JasonLG1979** [comment]
Sure. Put it after the [Make Raspotify use tmpfs for /tmp](https://github.com/dtcooper/raspotify/wiki/Basic-Setup-Guide#make-raspotify-use-tmpfs-for-tmp) section in the Basic Setup Guide.

**Comment by JavaMiata** [comment]
Finally got around to doing this now.  https://github.com/dtcooper/raspotify/wiki/Basic-Setup-Guide#ufw-settings

**Reply by JasonLG1979** [reply]
If you're going all in may as well do it right with an app profile:

https://askubuntu.com/questions/409013/how-do-you-create-an-app-profile-for-ufw

**Reply by JavaMiata** [reply]
If you want to add one, go ahead, I like to keep it simple with just whatever ports you need.  That way its obvious whats going on, and if someone has a different firewall setup they know what they need to do.

**Reply by JasonLG1979** [reply]
In a perfect world Raspotify/librespot ufw setting would be by app name and protocol and live somewhere in this repo and then at install/update time the configuration would just be automatic. I would like to do the same with apparmor.

**Reply by JasonLG1979** [reply]
> If you want to add one, go ahead,

I will need to do some testing. I don't use ufw, all my firewall setting live in my modem and router. I guess I know what I'm doing this weekend,lol!!!
